README.md

```diff
@@ -35,7 +37,8 @@ Convert the Ruby and ActiveRecord code below into Javascript and Mongoose code:
 ```
 
 ```js
-// Your answer...
+let andy = Instructor.findOne({name: "Andy"});
```

Comment:
Because mongo/mongoose operations are async, we need to work on the results with callbacks or promises:
```js
// Assuming we have model for wishlist_items, named WishItem
Instructor.findOne({name: "Andy"}, (andy)=>{
    andy.wishlist_items.push(new WishItem({description: "Resin Laying Deer Figurine, Gold"}))
    andy.save();
}
```